scripts/spec_xml/generate_spec_xml.py

```diff
@@ -77,12 +76,9 @@ def make_asciidoc(target: str, include_in_progress: str, spec_dir: str, dry_run:
     default=False,
     is_flag=True,
     help='Flag for dry run')
-@click.option(
```

Comment:
I don't think this is the same: this defines what we include from the spec. Please do not remove this one (but the rest could be removed)